tests/wsc/pom.xml

```diff
@@ -19,19 +19,20 @@
         <dependency>
             <groupId>org.mule.modules</groupId>
             <artifactId>mule-module-wsc</artifactId>
-            <version>4.0-SNAPSHOT</version>
+            <version>${parent.version}</version>
+            <classifier>mule-plugin</classifier>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.mule.modules</groupId>
             <artifactId>mule-module-cxf</artifactId>
-            <version>4.0-SNAPSHOT</version>
+            <version>${parent.version}</version>
```

Comment:
wrong. There won't be a release of the compatibility module pper each runtime release. Second, remember that plugins release cycles are going to be decoupled from the runtime release cycle so this won't always hold